goaway.go

```diff
@@ -92,12 +92,19 @@ func (g *ProfanityDetector) WithSanitizeSpaces(sanitize bool) *ProfanityDetector
 // WithCustomDictionary allows configuring whether the sanitization process should also take into account
 // custom profanities, false positives and false negatives dictionaries.
 func (g *ProfanityDetector) WithCustomDictionary(profanities, falsePositives, falseNegatives []string) *ProfanityDetector {
-	g.profanities = profanities
+	g.profanities = formatProfanities(profanities)
 	g.falsePositives = falsePositives
 	g.falseNegatives = falseNegatives
 	return g
 }
 
+func formatProfanities(s []string) []string {
+	for i, word := range s {
+		s[i] = strings.ToLower(word)
+	}
+	return s
+}
```

Comment:
should probably do it on all dictionaries

Reply:
Good call have changed the `formatProfanities`  function into `formatCustomDictionaries` and broke the logic of actually ToLower the slice into a function called `toLowerSlice`